docs/getting-started.md

```diff
@@ -82,8 +82,10 @@ kernel image with a Ubuntu 22.04 rootfs from our CI:
 ```bash
 ARCH="$(uname -m)"
 
+latest=$(wget "http://spec.ccfc.min.s3.amazonaws.com/?prefix=firecracker-ci/v1.9/x86_64/vmlinux-5.10&list-type=2" -O - 2>/dev/null | grep "(?<=<Key>)(firecracker-ci/v1\.9/x86_64/vmlinux-5\.10\.[0-9]{3})(?=</Key>)" -o -P)
```

Comment:
I'm afraid that will switch the problem from being "we forgot to update the patch version" to "we forgot to update the Firecrakcer version number in this command line". Is there a way to not hardcode it?

Reply:
We already have a script that updates the version string across the entire repository whenever we do a release. That should catch this